src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -78,6 +79,12 @@ class OverviewRoute extends React.Component<OverviewRouteProps, State> {
       <>
         <Row>
           <Col sm={colNum}>
+            {showMarketInsights && (
```

Comment:
For now, we should make this admin only. Once we're satisfied that it looks good on the artist page and are ready to launch the AB test we can prepare a PR that swaps out the admin only guard w/ the correct AB stuff.

Also, we probably want to have an accompanying storybook for this component. So it might be easier to leave out any artist page integration in this PR, and just have this PR be the new component and storybook/tests. Then we can get easy visual QA in storybooks.

Then, we can do an admin-only guard and add it to the artist page in a later PR, and then after that we would need that last PR to remove the admin-only guard in favor of the AB test.

It may sound a bit tedious but I think it's better to get into a rhythm of small/iterative PR's, vs one big feature launch.

That might be overkill however, so I'll leave it up to you!

Reply:
> Then, we can do an admin-only guard and add it to the artist page in a later PR, and then after that we would need that last PR to remove the admin-only guard in favor of the AB test.

That sounds great to me :+1: